### DISCUSSION TITLE
C# TimeSpan.Parse issue (25 hours -> 25 days)

### DISCUSSION THREAD

**Discussion by NikoUmbleLSPA** [dotnet/csharplang#8060] [General]
Hello, I'm struggling to figure out where to register a bug I found with C#, so here goes. When parsing a 25-hour string with TimeSpan.Parse, it instead parses out to 25 days.

I'm using Microsoft .NET Framework
Version 4.8.09032

![image](https://github.com/dotnet/csharplang/assets/161494325/00676515-d74c-435f-9476-502922a74cdf)

**Comment by CyrusNajmabadi** [comment]
Please file this at dotnet/runtime.  I unfortunately can't seem to move this issue there for some reason with GitHub that I haven't been able to figure out.

**Reply by NikoUmbleLSPA** [reply]
I've created https://github.com/dotnet/runtime/issues/101187. Thanks for the assistance.

**Comment by huoyaoyuan** [comment]
https://github.com/dotnet/runtime/issues/40316 https://github.com/dotnet/runtime/issues/90214

**Reply by NikoUmbleLSPA** [reply]
Much appreciated
